tools/generators/migrate-converged-pkg/index.ts

```diff
@@ -388,6 +393,43 @@ const templates = {
     .git*
     .prettierignore
   ` + os.EOL,
+  swcConfig: () => {
+    return {
+      $schema: 'https://json.schemastore.org/swcrc',
+      env: { targets: { chrome: '84', edge: '84', firefox: '75', opera: '73', safari: '14.1' }, bugfixes: true },
+      exclude: [
+        '/testing',
+        '/**/*.cy.ts',
+        '/**/*.cy.tsx',
+        '/**/*.spec.ts',
+        '/**/*.spec.tsx',
+        '/**/*.test.ts',
+        '/**/*.test.tsx',
+      ],
+      jsc: {
+        parser: {
+          syntax: 'typescript',
+          tsx: true,
+          decorators: false,
+          dynamicImport: false,
+        },
+        externalHelpers: true,
+        transform: {
+          react: {
```

Comment:
It might be a good idea to set `runtime` to `classic` as we still support React 16, https://swc.rs/docs/configuration/compilation#jsctransformreactruntime

Reply:
[90079dc](https://github.com/microsoft/fluentui/pull/26527/commits/90079dc3b6ea3930edb58ea2839d666c2601de64)